### ISSUE TITLE
Error decoding Github App `privateKey`

### ISSUE THREAD

**Issue by norman-zon** [promhippie/github_exporter#185]
I can't get the helm chart (v3.0.0, app version 2.0.0)  to work with 

```
config:
    authType: app
    privateKeyKey: <mykey>
    existingSecret: <mysecret>
```

or 

```
config:
authType: app
  appId: "999999"
  installationId: "999999"
  privateKey: "LS0tLS...."
```

In case I set the `privateKey` as **base64 encoded**, as described in the [values.yaml](https://github.com/promhippie/charts/blob/a2ad13ae3d773aa8403891cddb87f34f343e7dcd/charts/github-exporter/values.yaml#LL168C21-L168C21), I get this error:

``` 
{"date":"20230323","go":"go1.20.2","level":"info","msg":"Launching GitHub Exporter","revision":"97483e4","ts":"2023-03-23T13:10:43.798194204Z","version":"2.0.0"} 
{"err":"open LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFB....WS0tLS0tCg==: file name too long","level":"error","msg":"Failed to read GitHub key","ts":"2023-03-23T13:10:43.798349184Z"}
```

If I set it as **unencoded** PEM I get:

```
{"date":"20230323","go":"go1.20.2","level":"info","msg":"Launching GitHub Exporter","revision":"97483e4","ts":"2023-03-23T13:04:47.648184453Z","version":"2.0.0"} 
{"err":"illegal base64 data at input byte 0","level":"error","msg":"Failed to read GitHub key","ts":"2023-03-23T13:04:47.648268903Z"}
```

Looks to me like the [contentOrDecode](https://github.com/promhippie/github_exporter/blob/97483e4b7e48e684eebbfd6e37e3d4b9d8ce8b92/pkg/action/server.go#L322) function tries to open the base64-encoded string as a file instead of decoding it.

**Comment by tboerger** [comment]
Thanks for the report, I need to check how I will change the logic to avoid this error.

**Comment by tboerger** [comment]
I have published the chart version 3.0.1 which comes with a fix on the exporter version 2.0.1.

**Comment by norman-zon** [comment]
Thank you very much for the super fast fix! Your work here is very much appreciated ❤️
